src/main/python/tensorframes/core.py

```diff
@@ -79,8 +92,9 @@ def _check_fetches(fetches):
         return [fetches]
     return fetches
 
-def _get_graph(fetches):
+def _get_graph(fetches, initial_variable=True):
```

Comment:
You are repeating the default variable across the code, due to the syntax of python. This is annoying if one needs to change the default. The recommended pattern in python is the following:

```
_initial_variables_default = True

def _internal(initial_variables = None):
  if initial_variables is None:
    initial_variables = _initial_variables_default
  ...

def public(initial_variables = None):
  _internal(initial_variables)
```